pkgs/by-name/co/colorized-logs/package.nix

```diff
@@ -0,0 +1,35 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, stdenv
+, ...
+}:
+
+let
+  pname = "colorized-logs";
+  owner = "kilobyte";
+  version = "2.6";
+in
+
```

Comment:
Don't try to show expertise in language features.

```suggestion

```

Reply:
> Don't try to show expertise in language features.

I feel like this unnecessarily misconstrues what I did as having the sole intent of showing off.